notebooks/en/agent_change_llm.ipynb

```diff
@@ -0,0 +1,317 @@
+{
```

Comment:
<p>"We directly pass the initialized <code>HfEngine</code> to the <code>llm_engine</code> init of the agent."</p><p> </p><p>"This callable is the heart of the LLM engine."</p><p> </p><p>I don't think you need to nest those bullet points.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/140/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>